src/homework3/Car/Car.java

```diff
@@ -0,0 +1,92 @@
+package homework3.Car;
+
+import com.sun.org.apache.xpath.internal.SourceTree;
+
+public class Car {
+    private final String dateOfBurn;
+    private String typeOfEngine;
+    private int maxSpeed;
+    private int timeOf100km;
+    private int peopleCapacity;
+    private int hasPeople;
+    private int currentSpeed;
+    private CarWheel[] allWheel;
+    private CarDoor[] allDoor;
+
+    public Car(String dateOfBurn) {
+        this.dateOfBurn = dateOfBurn;
+    }
+
+    public Car(String dateOfBurn, String typeOfEngine, int maxSpeed, int timeOf100km, int peopleCapacity, int hasPeople, int currentSpeed) {
+        this.dateOfBurn = dateOfBurn;
+        this.typeOfEngine = typeOfEngine;
+        this.maxSpeed = maxSpeed;
+        this.timeOf100km = timeOf100km;
+        this.peopleCapacity = peopleCapacity;
+        this.hasPeople = hasPeople;
+        this.currentSpeed = currentSpeed;
+    }
+
+    public void setCurrentSpeed(int currentSpeed) {
+        this.currentSpeed = currentSpeed;
+    }
+
+    public void addOnePeople() {
+        if (peopleCapacity != hasPeople) hasPeople++;
+    }
+
+    public void minusOnePeaole() {
+        if (hasPeople != 0) hasPeople--;
+    }
+
+    public void allInPEople() {
+        hasPeople = 0;
+    }
+
+    public CarDoor getCarDoor(int i) {
+        try {
+            return allDoor[i];
+        }
+        catch (Exception e) {
+            System.out.println("Нет такой двери");
+            return null;
+        }
+    }
+
+    public void removeAllWheel() {
+        allWheel = new CarWheel[0];
+    }
+
+    public void addNewWheel(int n) {
+        CarWheel[] carWheels = allWheel;
+        allWheel = new CarWheel[carWheels.length + n];
+        for (int i = 0; i < carWheels.length; i++) {
+            allWheel[i] = carWheels[i];
+        }
+        for (int i = carWheels.length; i < allWheel.length ; i++) {
+            allWheel[i] = new CarWheel();
+        }
+    }
+
+    public int carrentMaxSpeed() {
+        if(hasPeople ==0) return 0;
```

Comment:
https://mate-academy.github.io/style-guides/java/java.html#s4.1.1-braces-always-used
Braces are used with if, else, for, do and while statements, even when the body is empty or contains only a single statement.